### PR TITLE
Bugfix/singlequbitlabelling

### PR DESCRIPTION
**Pull request by cqc-melf**
fix issue with unlabelled qubits only containing single qubit gates

### CODE REVIEW

File: pytket/binders/mapping.cpp [hunk 1]
```diff
@@ -149,7 +149,10 @@ PYBIND11_MODULE(mapping, m) {
           "qubits.\n\n:param circuit: pytket circuit to be mapped"
           "\n:param routing_methods: Ranked methods to use for routing "
           "subcircuits. In given order, each method is sequentially checked "
-          "for viability, with the first viable method being used.",
-          py::arg("circuit"), py::arg("routing_methods"));
+          "for viability, with the first viable method being used."
+          "\n:param label_single_qubits: will not label qubits without gates "
```
Comment (cqc-alec) [inline pytket/binders/mapping.cpp:153]: Maybe `label_isolated_qubits` would be a better name?

File: tket/src/Mapping/MappingManager.cpp [hunk 2]
```diff
@@ -100,6 +101,56 @@ bool MappingManager::route_circuit_with_maps(
...
+
+    node_vector_t nodes_vec = this->architecture_->get_all_nodes_vec();
+
+    // place all unplaced qubits
+
+    for (Qubit q : q_vec) {
+      if (!this->architecture_->node_exists(Node(q))) {
+        // found unplaced qubit
+        // other checks could be added here to avoid placing unused qubits or
+        // qubits that are not in an ppb
```
Comment (cqc-alec) [inline tket/src/Mapping/MappingManager.cpp:138]: ppb?

File: pytket/binders/mapping.cpp (hunk 1 above)
Comment (cqc-melf) [reply pytket/binders/mapping.cpp]: done

File: tket/src/Mapping/MappingManager.cpp (hunk 2 above)
Comment (cqc-melf) [reply tket/src/Mapping/MappingManager.cpp]: old comment, removed it

**Pull request merged**